### PR TITLE
new cog announce

### PR DESCRIPTION
**Pull request by brenfan**

### CODE REVIEW

File: announce.py [hunk 1]
```diff
@@ -0,0 +1,39 @@
...
+from discord.ext import commands
+import random as r
+import datetime
+
+class Announce:
+
+    def __init__(self, bot):
+        self.bot = bot
+
+    @commands.command(pass_context = True)
```
Comment (henrymzhao) [inline announce.py]: looks like you're trying to make the bot announce user's last msg then delete the user's msg? This is just gonna encourage spam.

Comment (brenfan) [inline announce.py]: I want to use it to format emails in #announce

Comment (henrymzhao) [inline announce.py]: Then you best lock it to the announce channel only :O

**Comment by brenfan** [comment]
ok added constraints

**Pull request merged**